docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicyUsage.md

```diff
@@ -174,7 +214,27 @@ Accept wildcard characters: False
 ```
 
 ### -Server
-{{Fill Server Description}}
+Specifies the Active Directory Domain Services instance to connect to, by providing one of the following values for a corresponding domain name or directory server.
+The service may be any of the following:  Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory snapshot instance.
+
+Specify the Active Directory Domain Services instance in one of the following ways:  
```

Comment:
```suggestion
Specifies the Active Directory Domain Services instance in one of the following ways:  
```

Reply:
This one should stay Specify as it should be read "You specify the Active..." I believe.